src/back/deps/svetit/shared/db/db_base.cpp

```diff
@@ -0,0 +1,43 @@
+#include "db_base.hpp"
+#include <memory>
+
+namespace svetit::db {
+
+Base::Base(storages::postgres::ClusterPtr pg)
+	: _db{std::move(pg)} {}
+
+Base::Base(storages::postgres::ClusterPtr pg, storages::postgres::Transaction&& trx)
+	: _db{std::move(pg)}
+	, _trx{std::make_shared<storages::postgres::Transaction>(std::move(trx))} {}
+
```

Comment:
Добавь пожалуйста деструктор
```suggestion

Base::~Base()
{
	if (_trx)
		_trx->Rollback();
}

```